### PR TITLE
draft add fhir preprocessing guide

### PR DESCRIPTION
**Pull request by tuvaforrest**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ const sidebars =
         "guides/claims-mapping",
         "guides/cms-hcc-risk-analytics",
         "guides/geospatial-analytics",
+        "guides/fhir-preprocessing",
```
Comment (aneiderhiser) [inline sidebars.js:67]: Let's change the order of the guides so that they are alphabetical

File: docs/guides/fhir-preprocessing.md [hunk 2]
```diff
@@ -0,0 +1,970 @@
+---
+id: fhir-preprocessing
+title: "FHIR Preprocessing"
+description: This guide walks you through how to transform FHIR data so its 
+---
+
+## Overview
+
+Fast Healthcare Interoperability Resources (FHIR) is a standard for health care data exchange.
```
Comment (aneiderhiser) [inline docs/guides/fhir-preprocessing.md:9]: healthcare spelled one word

File: docs/guides/fhir-preprocessing.md [hunk 3]
```diff
@@ -0,0 +1,970 @@
...
+description: This guide walks you through how to transform FHIR data so its 
+---
+
+## Overview
+
+Fast Healthcare Interoperability Resources (FHIR) is a standard for health care data exchange. 
+FHIR data typically comes in complex, nested JSON formats, making it challenging to process and analyze effectively. 
+Data teams are often more comfortable and productive working with data in familiar, tabular formats. 
+Preprocessing converts this complex data into a more manageable format, facilitating easier data analysis and integration. Our tool simplifies this process, enhancing the usability of FHIR data in healthcare analytics.
+
```
Comment (aneiderhiser) [inline docs/guides/fhir-preprocessing.md:13]: Add a very simple Miro diagram that shows the data transformation steps
- Start: Raw FHIR as JSON
- FHIR Inferno (flattens JSON to CSVs)
- Load CSVs into Data Warehouse
- Transform CSVs into Tuva Input Layer
- Run Tuva
- End: Analytics-ready Data

File: docs/guides/fhir-preprocessing.md [hunk 4]
```diff
@@ -0,0 +1,970 @@
...
+### Introduction to fhir-inferno:
+[FHIR Inferno](https://github.com/tuva-health/FHIR_connector) is a Python-based utility designed for efficient FHIR data transformation. The core functionality focuses on flattening FHIR data, creating structured tables suitable for analysis and reporting. In this guide we walk through the basic setup, configuration, and of utilization process of tool.
+
```
Comment (aneiderhiser) [inline docs/guides/fhir-preprocessing.md:16]: In this section we want to not only describe fhir inferno but the other components used in FHIR PP, namely HG connector and Tuva repo.

File: docs/guides/fhir-preprocessing.md [hunk 5]
```diff
@@ -0,0 +1,970 @@
...
+
+## Getting Started
```
Comment (aneiderhiser) [inline docs/guides/fhir-preprocessing.md:18]: This section describes FHIR Inferno - let's rename it FHIR Inferno.  We need subsequent sections for HG Connector and Tuva.

File: docs/guides/fhir-preprocessing.md [hunk 6]
```diff
@@ -0,0 +1,970 @@
...
+                    base, ext = os.path.splitext(relative_path)
+                    modified_relative_path = f"{base}_{recursion_depth}{ext}"
+                    output_file_key = os.path.join('FHIR_Output', modified_relative_path)
+                else:
+                    output_file_key = os.path.join('FHIR_Output', relative_path)
+                output_file_key = output_file_key.replace(os.path.sep, '/')
+                s3_client.upload_file(file_path, output_bucket_name, output_file_key)
+```
+
+</details>
```
Comment (aneiderhiser) [inline docs/guides/fhir-preprocessing.md:970]: We need 2 sections below this: one for HG connector and one for running Tuva.  These can be shorter.

**Review by aneiderhiser — CHANGES_REQUESTED** [review]

File: docs/guides/fhir-preprocessing.md (hunk 3 above)
Comment (tuvaforrest) [reply docs/guides/fhir-preprocessing.md:13]: lol, I wanted to know if chat friend could make something more interesting.  I think it'll still be a few years before they take over the world or our jobs.

image

**Review by aneiderhiser — APPROVED** [review]

**Pull request merged**